doc/radosgw/compression.rst

```diff
@@ -10,6 +10,11 @@ using any of the existing compression plugins.
 .. note:: The Reef release added a :ref:`feature_compress_encrypted` zonegroup
    feature to enable compression with `Server-Side Encryption`_.
 
+Supported compression plugins include the following:
+
+* zlib
+* snappy
+* zstd
```

Comment:
Should not `lz4` be in this list?  I've seen other omissions of it in our docs, and I'm using it myself at the OSD level.

```
./doc/rados/configuration/bluestore-config-ref.rst:BlueStore supports inline compression using `snappy`, `zlib`, `lz4`, or `zstd`.
./doc/rados/operations/pools.rst:   :Valid Settings: ``lz4``, ``snappy``, ``zlib``, ``zstd`` 
./doc/man/8/radosgw-admin.rst:    Placement target compression algorithm (lz4|snappy|zlib|zstd).

```

Reply:
By God, I think you're right.

Accepted.

Reply:
Is my veracity that dumbfounding? ;)